CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # App Center SDK for Android Change Log
 
-## Version 2.3.1 (Under development)
+## Version 2.3.1
```

Comment:
Do not remove (Under development), release team will do.
Since the changelog is visible on develop and that is the default branch it could give inaccurate info that the version is available.